contracts/BancorX/BancorX.sol

```diff
@@ -2,13 +2,12 @@
 pragma solidity 0.8.6;
 
 
-import "./interfaces/IBancorX.sol";
-
-import "./utility/TokenHolder.sol";
-
-import "./token/SafeERC20Ex.sol";
+import "./Utils.sol";
```

Comment:
I'm a bit conflicted about this external module that is only used by this specific contract. Perhaps we should eventually have our own utils module, and then we should use it all over our contracts (which means paying for audit changes), and perhaps we should get rid of the utils module and move the functions we need into the contract itself. At the very least we should make sure that we remove any unused functions from utils and change its name to "BridgeUtils" or something.

Reply:
In Utils there are modifiers. all of them are being used in BancorX.sol.

We can just define them in BancorX.sol and remove the Utils.sol file, instead of creating utils module that we don't really need at this point.